aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/BaseHandlerStd.java

```diff
@@ -220,6 +223,28 @@ protected boolean isDBClusterStabilized(
         }
     }
 
+    protected boolean isDBClusterStabilizedAfterRemoveFromGlobalCluster(
+            final ProxyClient<RdsClient> proxyClient,
+            final ResourceModel model,
+            final String previousGlobalClusterIdentifier
+    ) {
+        final DBCluster cluster = fetchDBCluster(proxyClient, model);
+
+        if (!DBClusterStatus.Available.equalsString(cluster.status()))
+            return false;
+
+        try {
+            final DescribeGlobalClustersResponse globalClustersResponse = proxyClient.injectCredentialsAndInvokeV2(
+                    Translator.describeGlobalClustersRequest(previousGlobalClusterIdentifier),
+                    proxyClient.client()::describeGlobalClusters
+            );
+            final GlobalCluster globalCluster = globalClustersResponse.globalClusters().get(0);
+            return globalCluster.globalClusterMembers().stream().noneMatch(m -> m.dbClusterArn().equals(cluster.dbClusterArn()));
+        } catch (GlobalClusterNotFoundException ex) {
```

Comment:
We may need to handle other possible exceptions that describe call may throw in production especially the retriable ones like throttling, internal failures,...etc

Reply:
The rest of the exceptions will be handled by the common exception handler code.  Added a test to show that in the code explicitly